### DISCUSSION TITLE
Webradio Delete Problem

### DISCUSSION THREAD

**Discussion by R101G** [rern/rAudio#613] [Fixed Issues]
I decided to give the Radio France addon a try, and now when I try to remove the streams, I find that the delete function in Library only deletes the cover art.

Also the Fip Rock station picks up cover art from the RP stream. I am not surprised nobody has noticed :)

**Comment by klassikmann** [comment]
> Also the Fip Rock station picks up cover art from the RP stream. I am not surprised nobody has noticed :)

I noticed it last night.

**Reply by klassikmann** [reply]
Fixed in rAudio 1 • 20211114
